network-api/networkapi/wagtailpages/migrations/0032_update_wide_images.py

```diff
@@ -0,0 +1,63 @@
+# # Generated by Django 3.0.14 on 2021-08-10 18:19
+
+from django.db import migrations
+import wagtail.core.blocks
+import wagtail.core.blocks.static_block
+import wagtail.core.fields
+import wagtail.images.blocks
+import wagtailmedia.blocks
+
+
+from networkapi.wagtailpages.pagemodels.blog.blog import BlogPage
+from networkapi.wagtailpages.pagemodels.modular import ModularPage
+from networkapi.wagtailpages.pagemodels.primary import PrimaryPage
+
+
+def update_annotated_image_blocks(qs):
+    # Loop through all pages
+    for page in qs:
```

Comment:
```suggestion
    for page in qs:
        if hasattr(page, 'alias_of') and page.alias_of != None:
            # This is a page alias, rather than a real page
            continue
```